src/nft.vy

```diff
@@ -275,7 +275,8 @@ def __init__(name_: String[25], symbol_: String[5], base_uri_: String[80], name_
     _BASE_URI = base_uri_
 
     self._transfer_ownership(msg.sender)
-    self._change_minter_status(msg.sender, True)
+    self.is_minter[msg.sender] = True
+    log RoleMinterChanged(msg.sender, True)
```

Comment:
Why revert back this change?

Reply:
good question, something must have went wrong. fixed